tensorflow/python/ops/image_ops_impl.py

```diff
@@ -354,6 +354,14 @@ def random_flip_up_down(image, seed=None):
 
   Returns:
     A tensor of the same type and shape as `image`.
+
```

Comment:
Hey! :smile:  We already have an example for this function.

Reply:
oh oops my bad, I'll remove that